features/lima/info.yaml

```
@@ -2,7 +2,8 @@ description: "image suitable for using with [lima](https://lima-vm.io)"
 type: flag
```

```suggestion
type: platform
```

we should have always one 'platform' feature, previously this was kvm, but now as we replace it the 'lima' feature needs to be of that type.